queue/init.lua

```diff
@@ -45,6 +45,12 @@ queue = setmetatable({
     end
 })
 
+local ok, VERSION = pcall(require, 'queue-version')
```

Comment:
Well, for now there isn't any valid case when you can't require version file.

Reply:
It's more just in case. For example, some repositories use `make/cmake`, where you can forget to transfer this file to the installation location. Specifically, here it was possible to forget to add this file to `build.module` in the rockspec.
Well, or is it in case the user wants to transfer files to his project and forgets file with version
But I think it can be simplified without `pcall`